core/templates/pages/topics-and-skills-dashboard-page/templates/create-new-skill-modal.template.html

```diff
@@ -20,19 +20,21 @@ <h3>Create New Skill</h3>
         <[errorMsg]>
       </i>
     </span>
-    <div class="oppia-rule-details-header" style="margin-bottom: 0.5em; margin-top: 1em;">
+    <div class="oppia-rule-details-header protractor-test-open-concept-card" style="margin-bottom: 0.5em; margin-top: 1em;" ng-click="openConceptCardExplanationEditor()">
```

Comment:
Avoid inline styling here. Maybe add it to `oppia-rule-details-header` class.

Reply:
Done.

Reply:
Er, not done?

Reply:
@kevintab95 Sorry, done!